cypress/e2e/14-timeline/timeline.spec.ts

```diff
@@ -143,6 +143,37 @@ describe("Timeline", () => {
             });
         });
 
+        it("should set size and position to event toggle on expanded view source event", () => {
+            sendEvent(roomId);
+            cy.visit("/#/room/" + roomId);
+            cy.setSettingValue("showHiddenEventsInTimeline", null, SettingLevel.DEVICE, true);
+            cy.contains(".mx_RoomView_body .mx_GenericEventListSummary .mx_GenericEventListSummary_summary",
+                "created and configured the room.");
+
+            // Edit message
+            cy.get(".mx_RoomView_body .mx_EventTile").contains(".mx_EventTile_line", "Message").within(() => {
```

Comment:
If there can be multiple items (like I would expect for tiles) I suggest to always use `contains` instead of `get`.
Otherwise Cypress runs the contains test on the first result of `cy.get(".mx_RoomView_body .mx_EventTile")`.

```suggestion
            cy.contains(".mx_RoomView_body .mx_EventTile .mx_EventTile_line", "Message").within(() => {
```

Reply:
I was following other existing examples on `threads.spec.ts`.

Thanks for the info!

Reply:
> I was following other existing examples on `threads.spec.ts`.
> 
> Thanks for the info!

Most of the time you are lucky that only a single element matches.
But imho the tests are more reliable when using `cy.contains`.